app/controllers/medical_records_controller.rb

```diff
@@ -60,12 +59,12 @@ def destroy
 
   private
     # Use callbacks to share common setup or constraints between actions.
-    def set_medical_record
-      @medical_record = MedicalRecord.find(params[:id])
-    end
+  def set_medical_record
+    @medical_record = MedicalRecord.find(params[:id])
+  end
 
     # Only allow a list of trusted parameters through.
     def medical_record_params
-      params.require(:medical_record).permit(:patient_id, :patology, :habits, :current_state)
+      params.require(:medical_record).permit(:patient_id, :patology, :habits, :current_state, :notes)
     end
-end
+  end
```

Comment:
All files should be finished with an empty line.
```suggestion
  end
  
```